package.json

```diff
@@ -90,8 +93,6 @@
     "react-popper": "^2.3.0",
     "react-portal": "^4.2.2"
   },
-  "entry": "lib/index.js",
```

Comment:
Not sure what this line did?

Reply:
No idea. Probs something in the BM template that doesn't do anything.